app/helpers/twitter_intent.rb

```diff
@@ -0,0 +1,17 @@
+module ExercismWeb
+  module Helpers
+    module TwitterIntent
+      def twitter_share(opts = {})
```

Comment:
Adding the KW args would take away a feature I was considering important: have a 'clean' url if no parameters are given:

```ruby
def twitter_share(opts = {})
 ....
end

twitter_share(text: 'Go for it')
# => "http://twitter.com/intent/tweet?text=Go+for+it"
```

Having explicit parameters:
```ruby
def twitter_share(text: nil, url: nil)
 "#{TWITTER_URL}%s" % { text: text, url: url}.to_query
end

twitter_share(text: 'Go for it')
# => "http://twitter.com/intent/tweet?text=Go+for+it&url="
```
But is not a big deal, I can change it.